db/seeds.rb

```diff
@@ -80,6 +77,26 @@ def random_record_for_org(org, klass)
   end
 end
 
+# ----------------------------------------------------------------------------
+# Request Units
+# ----------------------------------------------------------------------------
+
+%w(pack pad flat).each do |name|
```

Comment:
From slack discussion, sounds like `pad` shouldn't be an example here.

Reply:
Changed to `box`.